### DISCUSSION TITLE
private.css seems not to work ?

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#265] [Q&A] [answered]
Hi, 
I'm now on 3.01 works perfect and the admin page is more structured.
But I try to use the private.css to zoom the live view .

/* zoom livestream */
preview-stream {
    transform: scale(1.5)
}

I expact that the video is 50% zoomed out, so that the the webcam view fit to the printed image detail .
I tried also 2.5 to have the effect bigger to see if its has any effect.

In the log I see that the private.css is used .

Disconnected from client Address(host='192.168.2.140', port=54808) (sseservice.py:282)
2024-05-22 17:51:31,207 [    INFO] 192.168.2.140:54809 - "GET / HTTP/1.1" 200 (h11_impl.py:477)
2024-05-22 17:51:31,261 [    INFO] 192.168.2.140:54809 - "GET /private.css HTTP/1.1" 200 (h11_impl.py:477) 

So what do I wrong ?

**Comment by Striker62** [comment]
to see the effect press F5 should work ?

**Reply by mgineer85** [reply, marked as answer]
Yes, just tested. You missed the # which selects for the ID of the element:

```
#preview-stream {
  background-size: cover; /* default: contain to avoid cropping */
  transform: scale(1.5); /* default: 1.0 to apply no zoom. 1.5 zooms in, smaller values than 1.0 usually make no sense */
}

```

**Reply by Striker62** [reply]
Ok thanks .
